Formula/gssh.rb

```diff
@@ -13,13 +14,13 @@ class Gssh < Formula
     sha256 "ef0becf43b6bfe4a1c1a3cffc27c5c01f338348273f66c7f4e3355e05f55b508" => :sierra
   end
 
-  depends_on java: "1.8"
+  depends_on "openjdk@11"
 
   def install
     ENV["CIRCLE_TAG"] = version
     system "./gradlew", "shadowJar"
     libexec.install "cli/build/libs/gssh.jar"
-    bin.write_jar_script libexec/"gssh.jar", "gssh", java_version: "1.8"
+    bin.write_jar_script libexec/"gssh.jar", "gssh", java_version: "11"
```

Comment:
java_version is not needed

Reply:
hmm, we dont have to specify `java_version` about which version to pick?

Reply:
only if it's a cask AFAIK

Reply:
that is right, yeah, this is compiled code, which already targets specific java binary format. Make sense.